packages/playground/resolve/index.html

```diff
@@ -107,6 +116,18 @@ <h2>resolve package that contains # in path</h2>
   import deepPath from 'resolve-exports-path/deep.json?url'
   text('.exports-deep-query', deepPath)
 
+  // deep import SVG w/ exports w/ query
+  import deepSvgPath from 'resolve-exports-path/deep.svg?url'
+  text('.exports-deep-svg-query', deepSvgPath)
+
+  // deep import HTML w/ exports w/ query
+  import deepHtmlPath from 'resolve-exports-path/deep.html?url'
+  text('.exports-deep-html-query', deepHtmlPath)
+
+  // deep import non-ESM w/ exports w/ query
+  import deepNonEsmPath from 'resolve-exports-path/deep-non-esm.js?url'
+  text('.exports-deep-non-esm-query', deepNonEsmPath)
```

Comment:
Without any `exports` defined, this kind of thing works.

But when `exports` is added, then you start running into errors like:

```
Uncaught SyntaxError: The requested module '/@fs/C:/project/asset-build/assets/main.js' does not provide an export named 'default'
```

Test-wise, everything just shows `fail` when the `import deepNonEsmPath from 'resolve-exports-path/deep-non-esm.js?url'` is in place here